### DISCUSSION TITLE
CefSharp.PostMessage

### DISCUSSION THREAD

**Discussion by Zer0bytte** [cefsharp/CefSharp#4744] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

121.3.130

### Operating System

Windows 10

### Architecture

x64

### .Net Version

.net 7

### Implementation

OffScreen

### Reproduction Steps

NA

### Expected behavior

how to use CefSharp.PostMessage in javascript is it deleted in that version?

### Actual behavior

Not Found

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

**Comment by amaitland** [comment]
In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask questions.

> how to use CefSharp.PostMessage in javascript is it deleted in that version?

What do you mean? If there's no javascript then you cannot use javascript methods.
